content/post/benchmarking-gnns/index.md

```diff
@@ -0,0 +1,249 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Benchmarking Graph Neural Networks"
+subtitle: ""
+summary: ""
+authors: [vijay-dwivedi, chaitanya-joshi, xavier-bresson]
+reading_time: true
+tags:
+- Deep Learning
+- Graph Neural Networks
+- Benchmark
+categories: []
+date: 2020-06-15T03:03:47+08:00
+lastmod: 2020-06-15T03:03:47+08:00
+featured: false
+draft: false
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder.
+# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
+image:
```

Comment:
You may consider adding a featured image (or re-using one of the images from the blog body as a feature here).

Reply:
It will be good to have, b/c naturally, we will like to feature the blog and paper highly on our website.

Reply:
Yeah, I didnt know about it. Will add here.